.travis.yml

```diff
@@ -0,0 +1,11 @@
+services:
+- docker
+before_install:
+- docker build --target test --tag my-test-image .
+script:
+- docker run -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e TRELLO_API_TOKEN=$TRELLO_API_TOKEN -e TRELLO_API_KEY=$TRELLO_API_KEY  my-test-image tests

```

Comment:
Shorthand when you're keeping the same env var names: `-e TRELLO_BOARD_ID -e TRELLO_API_TOKEN -e TRELLO_API_KEY`

Reply:
The argument `tests` at the end of the line is unused, right?